exercises/killer-sudoku-helper/description.md

```diff
@@ -31,7 +31,7 @@ In a 2-digit cage with a sum 10, there are 4 possible combinations:
 - 37
 - 46
 
-![https://media.githubusercontent.com/media/exercism/v3-files/main/julia/killer-sudoku-helper/example2.png](https://media.githubusercontent.com/media/exercism/v3-files/main/julia/killer-sudoku-helper/example2.png)
+![Sudoku grid, all squares empty except for the middle column, column 5, which has 8 rows filled. Each continguous two rows form a killer cage and are marked as grouped together. From top to bottom: first group is a cell with value 1 and a pencil mark indicating a cage sum of 10, cell with value 9. Second group is a cell with value 2 and a pencil mark of 10, cell with value 8. Third group is a cell with value 3 and a pencil mark of 10, cell with value 7. Fourth group is a cell with value 4 and a pencil mark of 10, cell with value 6. The last cell in the column is empty.](https://media.githubusercontent.com/media/exercism/v3-files/main/julia/killer-sudoku-helper/example2.png)
```

Comment:
May want to denote the highlight of 6? If it's not important, leave it out and this is good.

Reply:
The highlight is irrelevant, I have to update the screenshot 😬

Reply:
fixed in https://github.com/exercism/v3-files/commit/f2a263770feac472269db1ecc0f9faa76fb27d82